### PR TITLE
Transformations: Organise fields transformer fixes & detailing

### PR DESCRIPTION
**Pull request by dprokop**
Ref https://github.com/grafana/grafana/issues/23551

Fixes:
- Organise fields transformer persisted options(name) is not reflected in the organise transformer editor
- Organise transformer UI - long field names are not wrapped, we need to either wrap or add ellipsis+tooltip
- Table- header cell text overflows container - add ellipsis 
- Missing key prop on transformation cards

### CODE REVIEW

File: packages/grafana-ui/src/components/Table/styles.ts [hunk 1]
```diff
@@ -52,14 +53,18 @@ export const getTableStyles = stylesFactory(
...
 
         &:last-child {
           border-right: none;
         }
       `,
+      headerCellLabel: css`
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+      `,
```
Comment (torkelo) [inline packages/grafana-ui/src/components/Table/styles.ts:67]: I am sure you tried and this is needed, but can't this be on the headerCell class?

Comment (dprokop) [reply packages/grafana-ui/src/components/Table/styles.ts:67]: unfortunatelly no :( and yes, I've tried.

File: packages/grafana-ui/src/components/TransformersUI/OrganizeFieldsTransformerEditor.tsx [hunk 2]
```diff
@@ -133,12 +138,15 @@ const DraggableFieldName: React.FC<DraggableFieldProps> = ({
...
                 onClick={() => onToggleVisibility(fieldName, visible)}
               />
-              <span className={styles.name}>{fieldName}</span>
+              <span className={styles.name} title={fieldName}>
+                {fieldName}
+              </span>
             </div>
             <Input
               className="flex-grow-1"
+              defaultValue={renamedFieldName || ''}
```
Comment (torkelo) [inline packages/grafana-ui/src/components/TransformersUI/OrganizeFieldsTransformerEditor.tsx:147]: this is confusing, is this not the same as fieldName ?

Comment (dprokop) [reply packages/grafana-ui/src/components/TransformersUI/OrganizeFieldsTransformerEditor.tsx:147]: No, the original field name is displayed on the left side as the label, in the input we are showing the name that was configured.

Comment (torkelo) [reply packages/grafana-ui/src/components/TransformersUI/OrganizeFieldsTransformerEditor.tsx:147]: still do not understand, should it not be setting value={renamedFieldName} then or is this an uncontrolled input?

**Review by torkelo — APPROVED** [review]

File: packages/grafana-ui/src/components/TransformersUI/OrganizeFieldsTransformerEditor.tsx (hunk 2 above)
Comment (dprokop) [reply packages/grafana-ui/src/components/TransformersUI/OrganizeFieldsTransformerEditor.tsx:147]: @torkelo mind the change from onChange to onBlur. From performance reason I don't want the view to update on every keystroke. Having `value` requires `onChange` which would have to be a noop.

Comment (dprokop) [reply packages/grafana-ui/src/components/TransformersUI/OrganizeFieldsTransformerEditor.tsx:147]: https://github.com/grafana/grafana/pull/23812/files/c2a7e4a851623b49eda73aa92b0daedc018d4c32#diff-6a04ae10d31ea5d1f589ea8ac37af893R149

**Pull request merged**